tests/useClickOutsideListener.test.tsx

```diff
@@ -1,15 +1,19 @@
+import { fireEvent, render, waitFor, screen } from '@testing-library/react';
 import { renderHook, act } from '@testing-library/react-hooks';
-import { fireEvent } from '@testing-library/react';
 import { useClickOutsideListener, UseClickOutsideListenerOptions } from '../src';
+import { MutableRefObject } from 'react';
+import React from 'react';
```

Comment:
Can please merge this 2 react import lines ?

Reply:
Yes , of course

Reply:
It's done !